src/gl.rs

```diff
@@ -498,6 +498,7 @@ pub trait Gl {
     ) -> GLint;
 
     fn alias_point_size_range(&self) -> (GLfloat, GLfloat);
+    fn max_viewport_dims(&self) -> (GLint, GLint);
```

Comment:
Please document the return value, even if it's pointing to the relevant `getIntegerv` spec.